static/src/stylesheets/module/commercial/glabs/_hosted-thrasher.scss

```diff
@@ -0,0 +1,243 @@
+.creative--hosted-thrasher {
+    .hosted-thrasher__body {
+        position: relative;
+        height: 100%;
+        padding-top: 0;
+        display: block;
+
+        @include mq(tablet) {
+            .has-flex & {
+                display: flex;
+                flex-direction: row;
+                flex-wrap: nowrap;
+                align-items: stretch;
+            }
+        }
+
+        &.hosted-thrasher__body--1-elems > .hosted-thrasher--element {
```

Comment:
What do you mean by that? Should I take it outside of the `.creative--hosted-thrasher` in order to decrease the specificity?